<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>IPMIMANAGER</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="hostnameLabel">
      <property name="text">
       <string>Hostname:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="hostnameLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="persistentCheckBox">
      <property name="text">
       <string>Save Credentials</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="runPowerButton">
      <property name="text">
       <string>Run Power</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="powerComboBox">
      <item>
       <property name="text">
        <string>status</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>on</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>off</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cycle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>reset</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>diag</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>soft</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="runSensorButton">
      <property name="text">
       <string>Run Sensor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sensorComboBox">
      <item>
       <property name="text">
        <string>Sensor data</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="outputTextEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="credit">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Made with ❤️ by &lt;/span&gt;&lt;a href=&quot;https://github.com/lilxandrias/ipmimanager/&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline; color:#1d99f3;&quot;&gt;lilxandrias&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
